driver/configuration/provider_viper.go

```diff
@@ -82,6 +82,7 @@ const (
 	ViperKeyPKCEEnforcedForPublicClients   = "oauth2.pkce.enforced_for_public_clients"
 	ViperKeyLogLevel                       = "log.level"
 	ViperKeyCGroupsV1AutoMaxProcsEnabled   = "cgroups.v1.auto_max_procs_enabled"
+	ViperKeyAllScopesIncludedIfEmpty       = "strategies.include_all_scopes_when_empty"
```

Comment:
I think this config option should be called differently:

```suggestion
	ViperKeyGrantAllClientCredentialsScopesPerDefault       = "oauth2.client_credentials.default_grant_allowed_scope"
```